README.md

```diff
@@ -356,26 +356,127 @@ which is why it starts with `_`.
 
 see ***undocumented*** creating a [secret-stack](https://github.com/ssbc/secret-stack) plugin.
 
-## Undocumented API
-
-these methods exist, and are used for something important, but are not currently documented.
-
-* getAtSequence     - get message at a {id,sequence}
-* getVectorClock    - get the latest sequence for every feed held in log
-* latest            - get latest `{id,sequence,ts}` for given feed, id is the message hash.
-* latestSequence    - get latest sequence number for given feed
-* getLatest         - get the latest message for a given feed
-* progress          - get the progress state of index building
-* status            - get status information about index building
-* version           - current ssb-db version
-* queue             - append a message to write queue (without writing immediately)
-* post              - get called back when a message is appended
-* since             - observable of the current offset of underlying flumelog.
-* addUnboxer        - add a handler for unboxing (decrypting) messages
-* box               - encrypt a message content
+### getAtSequence ([id, seq], cb(err, msg))
+
+get a message at a given feed `id` with given `sequence`.
```

Comment:
maybe: "for a given feed id" instead of "at a given feed" or is my english bad? :-)

Reply:
no your english is better than mine